### PR TITLE
Add mappings translations and improve mappings commands

### PR DESCRIPTION
**Pull request by williambl**
- Removes `yarn` as an alias for `fabricv`
 - Adds mojmap to mappings commands
 - Adds mappings translation commands
 - Adds error messages to mappings commands when no mapping is found
 - Fixes mappings command title always saying 'Yarn' not the actual mappings name
 - Fixes removing 'next' button on mappings messages not working properly

### CODE REVIEW

**Review by KiriCattus — APPROVED** [review]
Apart from lacking the @author tag at the top of each new class of yours everything _looks_ fine to me, but I'm not that well versed in Kotlin to fully grasp some of this sorry

File: src/main/java/com/mcmoddev/mmdbot/modules/commands/info/CmdTranslateMappings.kt [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+                val member = (original.value as Pair<*, *>).second as Field
+
+                member.obfMergedName?.let { memberName -> parent.obfMergedName?.let { className -> target.getClassByObfName(className)?.getFieldByObfName(memberName) } }
+            }
+            original.value is Pair<*, *> && (original.value as Pair<*, *>).second is Method -> {
+                val parent = (original.value as Pair<*, *>).first as Class
+                val member = (original.value as Pair<*, *>).second as Method
+
+                member.obfMergedName?.let { memberName -> parent.obfMergedName?.let { className -> target.getClassByObfName(className)?.getMethodByObfName(memberName) } }
+            }
```
Comment (TheSilkMiner) [inline src/main/java/com/mcmoddev/mmdbot/modules/commands/info/CmdTranslateMappings.kt]: You might want to store `original.value` in a `val` to leverage smart casts and get rid of some of the noise

File: src/main/java/com/mcmoddev/mmdbot/modules/commands/info/CmdTranslateMappings.kt [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+            val targetMappings = namespace2.getProvider(version).get()
+            var hasPerfectMatch = false
+            var embeds = (MappingsQuery.queryClasses(
+                QueryContext(
+                    originProvider,
+                    query
+                )
+            ).value.asSequence() + MappingsQuery.queryMember(
+                QueryContext(originProvider, query)
+            ) { it.members.asSequence() }.value.asSequence())
```
Comment (TheSilkMiner) [inline src/main/java/com/mcmoddev/mmdbot/modules/commands/info/CmdTranslateMappings.kt]: There's a bit of inconsistent styling here

**Review by TheSilkMiner — APPROVED** [review]

File: src/main/java/com/mcmoddev/mmdbot/modules/commands/info/CmdTranslateMappings.kt (hunk 1 above)
Comment (williambl) [reply src/main/java/com/mcmoddev/mmdbot/modules/commands/info/CmdTranslateMappings.kt]: Thanks, fixed.

File: src/main/java/com/mcmoddev/mmdbot/modules/commands/info/CmdTranslateMappings.kt (hunk 2 above)
Comment (williambl) [reply src/main/java/com/mcmoddev/mmdbot/modules/commands/info/CmdTranslateMappings.kt]: Looks a little nicer now

**Review by jriwanek — APPROVED** [review]

**Review by KiriCattus — APPROVED** [review]

**Review by sciwhiz12 — APPROVED** [review]
Looks good to my not-so-Kotlin-knowing eyes.

**Comment by KiriCattus** [comment]
I have NO idea what these merge conflicts are sorry, github desktop won't show me and github itself won't show me either

**Comment by williambl** [comment]
> I have NO idea what these merge conflicts are sorry, github desktop won't show me and github itself won't show me either

Fixed

**Pull request merged**